script.js

```diff
@@ -1,147 +1,146 @@
-
-  })
-})
-.then((response) => {
-  if (response.status === 400){
-    throw new Error("Имя или текст короче 3 символов");
-  } 
-  if (response.status === 500) {
-    throw new Error("Кажется сервер сломался");
-  }
-
-  mainForm.style.display = "none";
-  loader.style.display = "flex";
-  return fetchAndLogComments();
-  })
-.then(() => {
-  mainForm.style.display = "flex";
-  loader.style.display = "none";
-  textInputElement.value = "";
-  textareaInputElement.value = "";
-})
-.catch((error) => {
-  if (error.message === "Кажется сервер сломался") {
-      mainForm.style.display = "flex";
-      loader.style.display = "none";
-    alert("Сервер упал");   
-    return;
-  }
-  if (error.message === "Имя или текст короче 3 символов") {
-      mainForm.style.display = "flex";
-      loader.style.display = "none";
-    alert("Имя или текст не могут быть короче 3 символов")  ;
-    return  ; 
-  }
-
-  mainForm.style.display = "flex";
-  loader.style.display = "none";
-  alert("Интернет соединение прервано, попробуйте позже");
-  return;
-})
-
-}
-
-
-
-// отправка комментарий нажатием Enter
-
-mainForm.addEventListener("keyup", (action) => {
-
-  if (action.code === "Enter" && isValidForm()) {
-    postComment();
- 
-}})
-
- // обработчик клика при нажатии на кнопку "написать"
-
-buttonComment.addEventListener("click", () => {
-
-  if (textInputElement.value === '' || textareaInputElement.value === '') {
-    if (textInputElement.value === '') 
-    textInputElement.classList.add('error');
-    if (textareaInputElement.value === '')
-      textareaInputElement.classList.add('error');
-    return;
-}
- postComment()
-
- textInputElement.classList.remove('error');
- textareaInputElement.classList.remove('error');
-
- renderComments();
-}) 
-
-// обработчик клика при нажатии на кнопку "удалить последний комментарий"
-
-removeComment.addEventListener("click", () => {
-    listElement.removeChild(listElement.lastElementChild);
-});
-
-
-answer();
-initAddLike();
-renderComments();
-
-
-console.log("It works!");
\ No newline at end of file
+import { renderLoginComponent } from './login-component.js';
+import { addComment, getComments, deleteComment} from './api.js';
+import { getListComments, initLikeListeners, initCommentListeners } from './export_function.js';
+import { siteView } from './siteView.js';
+
+
+const appEl = document.getElementById("app");
+let listComments = [];
+let token =null;
+let user = null;
+
+const updateComments = (firstLoading = false) => {
+
+  // Первая загрузка
+  if (firstLoading) {
+    renderComments(listComments, true);
+  }
+  // Чтение данных по API и обновление экрана
+  getComments(token).then((responseData) => {
+    const listComments= responseData.comments.map((comment) => getListComments(comment));
+
+
+    renderComments(listComments);
+  });
+};
+
+function postComment(TextAreaElement, InputElement) {

```

Comment:
По соглашению, названия переменных начинаются с маленькой буквы.
На функционал это не влияет, просто так не пишут:

```suggestion
function postComment(textAreaElement, inputElement) {

```

Reply:
Здесь и дальше по коду